index.html

```diff
@@ -1,18 +1,28 @@
 <!doctype html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <!-- https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP -->
-        <meta
-            http-equiv="Content-Security-Policy"
-            content="default-src 'self'; script-src 'self'"
-        />
-        <title>Hello World!</title>
-    </head>
-    <body>
-        <h1>Hello World!</h1>
-        We are using Node.js <span id="node-version"></span>, Chromium
-        <span id="chrome-version"></span>, and Electron
-        <span id="electron-version"></span>.
-    </body>
+<head>
+    <meta charset="UTF-8" />
+    <!-- https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP -->
+    <meta
+        http-equiv="Content-Security-Policy"
+        content="default-src 'self'; script-src 'self'"
+
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
+    <link href="styles.css" type="text/css" rel="stylesheet">
+    <title>Password Frog</title>
+</head>
+<body>
+    <div class="wrapper">
+        <form class="form">
+            <div class="form-group flex-grow-1 w-100">
+                <textarea class="form-control textarea" placeholder="Enter the text to encrypt" id="textToEncrypt"></textarea>
+            </div>
+            <button type="submit" class="btn btn-primary mt-2">Submit</button>
+        </form>
+    </div>    
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script>
```

Comment:
I don't understand, why do you get boostrap from cdn and download in `node_modules` simultaneously?

Reply:
@agoalofalife good point, deleted it from the html file. but as the node_modules are in gitignore, how to make sure that the library is included for everyone? I guess that i do not know how to manage the dependencies here.

Reply:
So, you have to point out this dependency in package.json - it might be fun but one of a few reasons npm exists
Colleagues have to make `npm install` and download all dependencies in node_modules folder

Reply:
@agoalofalife  it is already the case, so this topic is resolved. thanks!